.github/workflows/create-gui-pr.yml

```diff
@@ -79,14 +75,14 @@ jobs:
           token: ${{ steps.generate-token.outputs.token }}
           path: main-application
           commit-message: |
-            chore(gui): updates GUI
+            chore(deps): bump kuma-gui to ${{ github.sha }}
 
-            Updates GUI.
+            Bumps kuma-gui to version ${{ github.ref_name }}@${{ github.sha }}
           committer: GitHub <noreply@github.com>
           author: github-actions[bot] <github-actions[bot]@users.noreply.github.com>
           signoff: true
-          branch: chore/update-gui-${{ steps.date.outputs.date }}
+          branch: chore/update-gui-${{ github.sha }}
           delete-branch: true
-          title: "chore(gui): updates GUI (${{ steps.date.outputs.date }})"
-          body: "Update GUI."
+          title: "chore(deps): bump kuma-gui"
```

Comment:
**Suggestion**: add `skip ci` hint to skip CI for these PRs (see https://circleci.com/docs/skip-build).

```suggestion
          title: "chore(deps): bump kuma-gui [skip ci]"
```

Reply:
Can we put it in the body instead? It's a little noisy otherwise

Also should we add a `> Changelog: updateGUI` in the body so that changelog doesn't get too messy?

Reply:
Should we maybe just skip the Changelog entirely here?

Reply:
Yeah, I don’t see a benefit in having a changelog entry for this unless we actually compile the GUI changelog into this PR so that you actually know what changed. As far as troubleshooting goes, the git history will be good enough to see when one of these PRs was merged (though searching one markdown file might be quicker technically).

Reply:
Ok so `> Changelog: skip`

Reply:
That's a good point though, that we probably want to include relevant `kuma-gui` changes into the changelog we publish